services/dynamic-sidecar/requirements/_base.txt

```diff
@@ -328,11 +328,11 @@ opentelemetry-instrumentation-fastapi==0.49b2
     # via -r requirements/../../../packages/service-library/requirements/_fastapi.in
 opentelemetry-instrumentation-httpx==0.49b2
     # via -r requirements/../../../packages/service-library/requirements/_fastapi.in
-opentelemetry-instrumentation-logging==0.48b0
+opentelemetry-instrumentation-logging==0.49b2
```

Comment:
there are a few updates I don't quite get why they come here. are you sure about these?